vaxrank/report.py

```diff
@@ -308,18 +308,13 @@ def compute_template_data(self):
 
                 epitopes = []
                 wt_epitopes = []
-                sorted_epitope_predictions = sorted(
-                    vaccine_peptide.epitope_predictions, key=attrgetter('ic50'))
-                for epitope_prediction in sorted_epitope_predictions:
-                    epitope_data = self._epitope_data(epitope_prediction)
-                    # mutant vs WT epitope data
-                    if epitope_prediction.overlaps_mutation:
-                        epitopes.append(epitope_data)
-                    else:
-                        # only care about allele, ic50, sequence (other info is redundant)
-                        key_list = ['Allele', 'IC50', 'Sequence']
-                        wt_epitopes.append({key: epitope_data[key] for key in key_list})
+                for mutant_epitope_prediction in vaccine_peptide.mutant_epitope_predictions:
+                    epitopes.append(self._epitope_data(mutant_epitope_prediction))
```

Comment:
I think I've missed this from pervious PRs, but what does `_epitope_data` do?

Reply:
There are a bunch of helper functions to this effect; it makes an OrderedDict from an epitope prediction that then goes in as a template variable.